benchmarks/torchbench/reqs/setup.py

```diff
@@ -0,0 +1,49 @@
+# Fake setup.py to be used with piptools compile
```

Comment:
Why is this needed? I don't fully understand what's going on in this file.

Reply:
This is used in benchmarks/torchbench/reqs/pip-compile.sh, which is the script to build the reqs for all benches, and instead of writing a longer more complex bash script, I've offloaded some of the logic for fix some issues in python through those fake setup.py